_install-and-configure/install-opensearch/index.md

```diff
@@ -77,6 +77,13 @@ vm.max_map_count=262144
 
 Then run `sudo sysctl -p` to reload.
 
+For Windows workloads, you can set the `vm.max_map_count` running the following commands:
```

Comment:
Would this make more sense on the Window's page? https://opensearch.org/docs/latest/install-and-configure/install-opensearch/windows/

Reply:
you may be right, i will create a PR soon